project-api/project-api-test/src/test/java/io/ballerina/projects/test/BaseTest.java

```diff
@@ -91,6 +92,13 @@ protected void cacheDependencyToLocalRepository(Path dependency) throws IOExcept
         BaseTest.this.cacheDependencyToCentralRepository(dependencyProject, LOCAL_REPOSITORY_NAME);
     }
 
+    protected void cacheDependencyToLocalRepository(Path dependency,
+                                                    ProjectEnvironmentBuilder projectEnvironmentBuilder)
+            throws IOException {
+        BuildProject dependencyProject = TestUtils.loadBuildProject(projectEnvironmentBuilder, dependency);
+        BaseTest.this.cacheDependencyToCentralRepository(dependencyProject, LOCAL_REPOSITORY_NAME);
```

Comment:
Shall we change the method name `cacheDependencyToCentralRepository` to `cacheDependencyToRepositories` since we use the same method for caching to the local repo?

Reply:
Changed to `cacheDependencyToRepositories`